index.html

```diff
@@ -64,28 +72,36 @@ <h2>Software Developer</h2>
   </div>
   <div class="devider-skill"><hr></div>
   <!-- Social Contact list -->
-  <div class="email-icon">
+  <div class="work-arrow">
+    <div class="bar">
+      <img src="imgs/arrow.svg" alt="arrow" />
+      <h3>work</h3>
+    </div>
     <ul class="icon">
       <li><a href="https://github.com/GraceAbike"><img src="imgs/Vector.png" alt="Github"></a></li>
       <li><a href="https://www.linkedin.com/in/grace-abike-02770522a/"><img src="imgs/Icon-Linkedin.png" alt="LinkedIn"></a></li>
       <li><a href="https://angel.co/u/grace-abike"><img src="imgs/angelist-1.png" alt="Angelist"></a></li>
       <li><a href="https://twitter.com/GraceAbike1"><img src="imgs/Icon-Twitter.png" alt="Twitter"></a></li>
       <li><a href="#"><img src="imgs/Icon-Medium.png" alt="Medium"></a></li>
     </ul>
+    <div class="bar">
+      <img src="imgs/arrow.svg" alt="arrow" />
+      <h3>work</h3>
+    </div>
   </div>
   </section>
   
   <!-- The first Of Portfolio Mobile Card -->
   <section id="work-section">
-    <div class="work-section">
+    <div class="work-section work-section1">
       <img src="imgs/arrow.svg" alt="down-arrow">
       <h3>Work</h3>
     </div>
     <div class="work">
       <h2>MY RECENT<br>WORK</h2>
       <div class="devider"><hr></div>
     </div>
-    <div class="background-card1">
+    <div class="background-card1 background-card">
```

Comment:
[Optional] I suggest that you stick to the casing as given by the design. Also, separate the word _recent_ from the word _work_. Check the comment above for more on why you should follow a given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/186681969-ec48c4a6-2473-40aa-90df-3ef4a4d158e0.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/186682043-9a7043db-6688-4d81-b2e8-bddf9689781a.png)